docs/data-sources/dms_rabbitmq_flavors.md

```diff
@@ -0,0 +1,143 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_rabbitmq_flavors
+
+Use this data source to get the list of rabbitMQ available flavor details within HuaweiCloud.
+
+## Example Usage
+
+### Query the list of rabbitmq flavors by cluster type
```

Comment:
s/rabbitmq/rabbitMQ
MQ is the abbreviation of message queue

Reply:
done